src/main/scala/wdl4s/WdlNamespace.scala

```diff
@@ -222,7 +227,7 @@ object WdlNamespace {
 
     def getScope(scopeAst: Ast, parent: Option[Scope]): Scope = {
       val scope = scopeAst.getName match {
-        case AstNodeName.Call => Call(scopeAst, namespaces, tasks, wdlSyntaxErrorFormatter)
+        case AstNodeName.Call => Call(scopeAst, namespaces, topLevelTasks, wdlSyntaxErrorFormatter)
```

Comment:
Am I missing something? Looks like the second `case AstNodeName.Call` will never hit if this one catches everything?